src/index.ts

```diff
@@ -28,18 +28,19 @@ export default class TiledResource extends Resource<ITiledMap> {
    protected mapFormat: TiledMapFormat;
 
    public imagePathAccessor: (path: string, ts: ITiledTileSet) => string;
+   public externalTilesetPathAccessor: (path: string, ts: ITiledTileSet) => string;
```

Comment:
Can you add examples of when you would use `imagePathAccessor` and/or `exteranalTilesetPathAccessor`?

Reply:
Better? 👍